agent/xds/delta.go

```diff
@@ -309,7 +305,7 @@ func (s *Server) processDelta(stream ADSDeltaStream, reqCh <-chan *envoy_discove
 						op.TypeUrl, err)
 				}
 				if sent {
-					sentType[op.TypeUrl] = struct{}{}
```

Comment:
If a reviewer expands the below and looks at `var xDSUpdateOrder` on line 319 that explains what this loop is looping over.